### PR TITLE
Fix broken links

### LINKED ISSUE #24: broken image links in tutorial
Hey folks,
Just trying to go through the tutorial and noticed that the first two images appear to be broken.

### PR DESCRIPTION
**Pull request by cameronmartino**
resolves #24 of broken image links.

- updates readme links
- updates in-repo tutorial links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ To install the most up to date version of deicode, run the following command
 
 ## Qiime2 tutorial
 
-**Note**: the official plugin docs and tutorial can be found [here](https://library.qiime2.org/plugins/deicode).
-
-* [Tutorial](https://github.com/biocore/DEICODE/blob/master/ipynb/tutorials/moving-pictures.md)
+* The Qiime forum tutorial can be found [here](https://forum.qiime2.org/t/robust-aitchison-pca-beta-diversity-with-deicode/8333).
```
Comment (ElDeveloper) [inline README.md]: Qiime -> QIIME2

**Review by ElDeveloper — APPROVED** [review]
One minor suggestion.

**Comment by cameronmartino** [comment]
@ElDeveloper should be good to go now. Thanks!

**Comment by ElDeveloper** [comment]
Thanks @cameronmartino

**Pull request merged**